### DISCUSSION TITLE
Version 3 Release Timeline and Offer to Contribute

### DISCUSSION THREAD

**Discussion by luizjr** [requarks/wiki#7314] [Help / Questions] [answered]
Hi there,

First of all, I want to commend the team for the incredible work on this project. I'm a developer and have been using this tool in my own projects. I've been following the updates and am particularly interested in the upcoming major release, version 3.

I’m reaching out to inquire if there’s an estimated timeline for the release of version 3 or if there’s already a development plan in place. Additionally, I’d love to contribute to the project. If there are any tasks, issues, or areas where you need assistance—whether it’s implementing new features, fixing bugs, or performing tests—I’d be happy to help.

I look forward to hearing more details and am eager to contribute!

Thank you!

Luiz Silva

**Comment by mihailim** [comment, marked as answer]
https://docs.requarks.io/en/releases
